src/main/java/katas/Kata6.java

```diff
@@ -14,6 +19,15 @@
     public static String execute() {
         List<Movie> movies = DataUtil.getMovies();
 
-        return "someUrl";
+        //return "someUrl";
+        return DataUtil.getMovies()
+                .stream()
+                .map(Movie::getBoxarts)
+                .flatMap(Collection::stream)
+                .collect(Collectors.toList())
+                .stream()
+                .reduce((x1, x2) -> (x1.getHeight() * x1.getWidth()) > (x2.getHeight() * x2.getWidth()) ? x1 : x2)
```

Comment:
[mf] pls extract this lambda into separate private method for better readability